pkg/server/testserver.go

```diff
@@ -426,7 +428,7 @@ func WaitForInitialSplits(db *client.DB) error {
 	if err != nil {
 		return err
 	}
-	return retry.ForDuration(initialSplitsTimeout, func() error {
+	err = retry.ForDuration(initialSplitsTimeout, func() error {
```

Comment:
You could log every time this retries. The max backoff here is 1s. If you don't see the message regularly, this can hint at starvation (perhaps due to other packages being tested in parallel).

Reply:
Good idea. I've noticed recently that `SucceedsSoon` does this.